src/web-socket/web-socket.ts

```diff
@@ -1,40 +1,24 @@
 import { webSocket } from "..";
```

Comment:
I don't want want you to waste your time too much on this but you should understand that as of now it seems that if someone will use your http/ws api the gql subscription will never update with the changes

Reply:
Yea 😒 ... du u have a solutuon? I set entire day on this